dockerfiles/artifactory/Dockerfile

```diff
@@ -4,8 +4,6 @@ MAINTAINER Snyk Ltd
 
 USER root
 
-RUN apt update && apt install -y make python gcc
```

Comment:
I believe `npm install` with `node-gyp` requires this tooling to be available specially during build stage, do we not need this anymore?